### ISSUE TITLE
Use widget to add recovery question on first login. 

### ISSUE THREAD

**Issue by chris-whittick-vgh** [okta/okta-signin-widget#182]
Add setting recovery question to the widget for new accounts.

Users are being created in Okta when logging into an old login system. When switching over to an Okta login using the widget they are redirected away from the login page to an Okta Page to select a security question. Would be nice to add this as part of the login page flow, the user experience because of the redirect creates a inconsistent experience.

**Comment by asharma93** [comment]
+1 👍

**Comment by rchild-okta** [comment]
Yep, definitely agree (and it's something we want to do). There is a bit of work on the API level that needs to be done before we can support this - recovery question is currently part of the registration flows (and not authn, which the sign-in widget is a wrapper of).

**Comment by StateBarofArizona** [comment]
Any update?  Can those screens at least be iframed to maintain some sort of consistency in the UI?

**Comment by chris-whittick-vgh** [comment]
Hey, it looks like in Early Access now you can optional use security questions to reset passwords (rather than required). This partially solves my issue I think since we reflect users on login from our legacy into Okta and generate a security QA at that time that we store locally.  Haven't tested it yet though

**Comment by milkolori** [comment]
Any update here? Is this functionality available thru OKta Login widget?
